src/xenia/gpu/primitive_processor.cc

```diff
@@ -823,6 +823,26 @@ bool PrimitiveProcessor::Process(ProcessingResult& result_out) {
     }
   }
 
+  // - 4D5307F1 - Continuous AutoIndexed QuadPatch draws initiated with only
+  // one vertex. host_draw_vertex_count needs to be rounded up to meet the
+  // requirements of the host hull shader.
+  if (tessellation_enabled &&
```

Comment:
For indexed draws (the R400 can tessellate regular primitives in discrete and continuous modes, not only the specialized patch primitive types), this is dangerous as indices beyond `guest_index_buffer_needed_bytes` not covered by `shared_memory_.RequestRange` will be fetched by the draw. Though I'm not sure how it should work on the real hardware in this case.